### DISCUSSION TITLE
Having trouble linking to other pages

### DISCUSSION THREAD

**Discussion by ghost** [mmistakes/minimal-mistakes#4541] [Q&A]
I'm struggling with this and could use some help. I've set up a GitHub page using this theme. I have the splash page set up the way that I want, but I get 404 errors whenever I click the link to other pages, which are saved as markdown files. When the markdown is empty, the page shows as whitespace when I click on the link, but as soon as I add any front matter, I get the 404 error. If I change the page from a .md to a .html, it works, but ideally I'd like to use the .md format.

What am I missing?

I've been doing most of my testing with the /about.md file. My plan was to figure that one out and then move from there.

Page URL: https://omi-masika.github.io/
Repository: https://github.com/omi-masika/omi-masika.github.io

**Comment by glennbroadway** [comment]
I had a similar issue and I used the "permalink:" in the front matter (is that what the settings at the top of each page are called?) to set a permalink that has neither .md or .html then I can link to the permalink.

e.g. permalink: /pages/page-name/
or permalink: /about/
